packages/keyring-controller/CHANGELOG.md

```diff
@@ -7,13 +7,19 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-## [19.2.1]
-
 ### Changed
```

Comment:
There are some changelog entries that have been misplaced: they are under 19.2.1 but they are actually unreleased